src/ace.F90

```diff
@@ -672,6 +681,7 @@ subroutine read_nu_data(nuc)
 
     else
       nuc % nu_d_type = NU_NONE
+      nuc % n_precursor = ZERO
```

Comment:
you are correct, `nuc % n_precursor` is an integer and should be set with `nuc % n_precursor = 0`.